Source/main.py

```diff
@@ -8,10 +8,35 @@
 import traceback
 import sys
 from pyboy import PyBoy
-from pyboy.logger import addconsolehandler
-
+from pyboy.logger import logger, addconsolehandler
 addconsolehandler()
 
+if "--no-logger" in sys.argv:
+    logger.disabled = True
+
+argv_debug = "--debug" in sys.argv
+argv_profiling = "--profiling" in sys.argv
+argv_autopause = "--autopause" in sys.argv
+argv_disable_input = "--no-input" in sys.argv
+
+# TODO: Find a library to take care of argv
```

Comment:
I have plans for this, if I can ever get around to it!

Reply:
Awesome! I'll leave it for you then. My idea with the new PyBoy constructor was also to make it easier to pass settings